src/app/utils/api/fetchMovies.ts

```diff
@@ -0,0 +1,7 @@
+import { Movie } from "@prisma/client";
+import { PaginationSchema } from "../paginationFilter";
+
+export const fetchMovies = async ({ limit, page }: PaginationSchema) => {
+  const result = await fetch(`/api/movies/all?limit=${limit}&page=${page}`);
```

Comment:
REST API convention for this endpoint is `/api/movies` + query string e.g. 

`/api/movies?limit=${limit}&page=${page}`

Reply:
Agreed, makes more sense this way.